tests/std/tests/P0896R4_views_transform_death/test.cpp

```diff
@@ -0,0 +1,355 @@
+    (void) (i + ranges::distance(some_ints)); // cannot seek transform_view iterator after end
+}
+
+void test_flipped_operator_plus_value_initialized() {
+    ranges::iterator_t<TV> i{};
+    (void) (42 + i); // cannot seek value-initialized transform_view iterator
+}
+
+void test_flipped_operator_plus_before_begin() {
+    TV r{some_ints, lambda};
+    ranges::iterator_t<TV> i = r.begin() + ranges::distance(some_ints) / 2;
+    (void) (-ranges::distance(some_ints) + i); // cannot seek transform_view iterator before begin
+}
+
+void test_flipped_operator_plus_after_end() {
+    TV r{some_ints, lambda};
+    ranges::iterator_t<TV> i = r.begin() + ranges::distance(some_ints) / 2;
+    (void) (ranges::distance(some_ints) + i); // cannot seek transform_view iterator after end
+}
+
+void test_operator_minus_value_initialized() {
+    ranges::iterator_t<TV> i{};
+    (void) (i - 42); // cannot seek value-initialized transform_view iterator
+}
+
+void test_operator_minus_before_begin() {
+    TV r{some_ints, lambda};
+    ranges::iterator_t<TV> i = r.begin() + ranges::distance(some_ints) / 2;
+    (void) (i - ranges::distance(some_ints)); // cannot seek transform_view iterator before begin
+}
+
+void test_operator_minus_after_end() {
+    TV r{some_ints, lambda};
+    ranges::iterator_t<TV> i = r.begin() + ranges::distance(some_ints) / 2;
+    (void) (i - -ranges::distance(some_ints)); // cannot seek transform_view iterator after end
+}
+
+void test_operator_minus_incompatible_different() {
+    TV r0{some_ints, lambda};
+    ranges::iterator_t<TV> i0 = r0.begin();
+    TV r1{some_ints, lambda};
+    ranges::iterator_t<TV> i1 = r1.begin();
+    (void) (i0 - i1); // cannot compare incompatible transform_view iterators
+}
+
+void test_operator_minus_incompatible_value_initialized() {
+    TV r{some_ints, lambda};
+    ranges::iterator_t<TV> i = r.begin();
+    (void) (i - ranges::iterator_t<TV>{}); // cannot compare incompatible transform_view iterators
+}
+
+void test_iter_move_value_initialized_iterator() {
+    ranges::iterator_t<TV> i{};
+    (void) ranges::iter_move(i); // cannot dereference value-initialized transform_view iterator
+}
+
+void test_iter_swap_value_initialized_iterators() {
+    ranges::iterator_t<TV> i0{};
+    ranges::iterator_t<TV> i1{};
+    (void) ranges::iter_swap(i0, i1); // cannot dereference value-initialized transform_view iterator
+}
+
+void test_iter_swap_value_initialized_iterator_left() {
+    ranges::iterator_t<TV> i0{};
+    TV r{some_ints, lambda};
+    ranges::iterator_t<TV> i1 = r.begin();
+    (void) ranges::iter_swap(i0, i1); // cannot dereference value-initialized transform_view iterator
+}
+
+void test_iter_swap_value_initialized_iterator_right() {
+    TV r{some_ints, lambda};
+    ranges::iterator_t<TV> i0 = r.begin();
+    ranges::iterator_t<TV> i1{};
+    (void) ranges::iter_swap(i0, i1); // cannot dereference value-initialized transform_view iterator
+}
+
+int main(int argc, char* argv[]) {

```

Comment:
Should there be positive test coverage (possibly in the non-death test) that seeking value-initialized iterators by zero, differencing value-initialized iterators, comparing value-initialized iterators, etc. behaves properly?

Reply:
Yes, I think I'm missing a lot of coverage of the iterator and sentinel operations.